src/common/svc/relay/create_raw_sub_spec.hpp

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+//
+
+#ifndef OCVSMD_COMMON_SVC_RELAY_CREATE_RAW_SUB_SPEC_HPP_INCLUDED
+#define OCVSMD_COMMON_SVC_RELAY_CREATE_RAW_SUB_SPEC_HPP_INCLUDED
+
+#include "ocvsmd/common/svc/relay/CreateRawSub_0_1.hpp"
+
+namespace ocvsmd
+{
+namespace common
+{
+namespace svc
+{
+namespace relay
+{
+
+struct CreateRawSubSpec
```

Comment:
class docs plz

Reply:
This is really internal IPC housekeeping specification for a service, but I will add a mention of this to the docs.